Add instructions on structuring url in poster exercise

jema28 opened the pull request.

jema28 committed:
add instructions on structuring url in poster ex

eadehemingway commented on README.md:
```diff
@@ -77,27 +77,110 @@ Status codes tell us if a request was successful. The main groups are:
-      Poster: "https://m.media-amazon.com/images/M/MV5BMjIyZGU4YzUtNDkzYi00ZDRhLTljYzctYTMxMDQ4M2E0Y2YxXkEyXkFqcGdeQXVyNTIzOTk5ODM@._V1_SX300.jpg"
-    }
-    ...
+1. Make an API call to OMDb movies (in `poster-gallery/script.js`) to get all Harry Potter films.
```
maybe change this to be 'the first task is to make an API call to OMDb movies to get all the harry potter information they have. To do this follow the steps below:

eadehemingway replied:
i think we shouldnt write 'harry potter movies' or 'harry potter movies and games' cos it might make them think they have to specify that somewehre?

jema28 replied:
Done 🐼

jema28 committed:
edit first point to poster-gallery exercise
- add note to follow instructions
- change mention of harry potter movies/films to all info.

jema28 committed:
change working in details/summary
